lib/erblint-github/linters/github/accessibility/no_aria_hidden_on_focusable_counter.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative "../../custom_helpers"
+
+module ERBLint
+  module Linters
+    module GitHub
+      module Accessibility
+        class NoAriaHiddenOnFocusableCounter < Linter
+          include ERBLint::Linters::CustomHelpers
+          include LinterRegistry
+
+          MESSAGE = "Elements that are focusable should not have `aria-hidden='true' because it will cause confusion for assistive technology users."
+
+          def run(processed_source)
+            tags(processed_source).each do |tag|
+              aria_hidden = possible_attribute_values(tag, "aria-hidden")
+              generate_offense(self.class, processed_source, tag) if aria_hidden.include?("true") && focusable?(tag)
```

Comment:
Is `aria_hidden.include?("true")` for `<button aria-hidden>Button</button>`?

Reply:
Nope! This does not treat `aria-hidden` like `aria-hidden="true"`. Just added a test for that.

I believe `aria-hidden` is not valid and an explicit attribute needs to be provided.

Reply:
> I believe aria-hidden is not valid and an explicit attribute needs to be provided.

Possibly a separate linter idea

Reply:
> I believe `aria-hidden` is not valid and an explicit attribute needs to be provided.

This is my understanding is well. 👍 Incidentally, I shared some related reading in a [thread in #accessibility-watercooler](https://github.slack.com/archives/C034UD4NRAS/p1675372437027389).